src/Canvas.cc

```diff
@@ -753,6 +754,7 @@ NAN_METHOD(Canvas::RegisterFont) {
       FontFace face;
       face.user_desc = user_desc;
       face.sys_desc = sys_desc;
+      strcpy( (char *)face.file_path, (char *) *filePath );
```

Comment:
`strncpy((char *)face.file_path, (char *) *filePath, 1023)`

Reply:
Ah, good catch.  I'll make that change tonight.  Thank you!

Reply:
Done!